server/utils/email/digest.tsx

```diff
@@ -44,6 +45,17 @@ const getAffectedObject = (item: ActivityItem, associations: ActivityAssociation
 	};
 };
 
+const getAffectedObjectUrl = (item: ActivityItem, associations: ActivityAssociations) => {
+	const community = Object.values(associations.community)[0];
+	if (item.pubId) {
+		return pubUrl(community, associations.pub[item.pubId]);
```

Comment:
In the case where a Pub was deleted , `associations.pub[item.pubId]` will be undefined — so we gotta check for the existence of that too before calling `pubUrl()`.